--- /local/comp3301/linux-3.9.4/fs/ext2/Makefile	2013-05-25 04:45:59.000000000 +1000
+++ Makefile	2013-08-18 12:57:57.133220160 +1000
@@ -1,13 +1,18 @@
 #
-# Makefile for the linux ext2-filesystem routines.
+# Makefile for the linux ext3301-filesystem routines.
 #
 
-obj-$(CONFIG_EXT2_FS) += ext2.o
+obj-m += ext3301.o
 
-ext2-y := balloc.o dir.o file.o ialloc.o inode.o \
+ext3301-y := balloc.o dir.o file.o ialloc.o inode.o \
 	  ioctl.o namei.o super.o symlink.o
 
-ext2-$(CONFIG_EXT2_FS_XATTR)	 += xattr.o xattr_user.o xattr_trusted.o
-ext2-$(CONFIG_EXT2_FS_POSIX_ACL) += acl.o
-ext2-$(CONFIG_EXT2_FS_SECURITY)	 += xattr_security.o
-ext2-$(CONFIG_EXT2_FS_XIP)	 += xip.o
+MOD_DIR=/local/comp3301/linux-3.9.4
+
+.PHONY: all
+all:
+	make -C $(MOD_DIR) M=$(PWD) ARCH=um modules
+
+.PHONY: clean
+clean:
+	make -C $(MOD_DIR) M=$(PWD) ARCH=um clean

